### PR TITLE
Add Honeyswap subgraph (Uniswap fork)

### PR DESCRIPTION
**Pull request by mrbrianhobo**
- Add Honeyswap subgraph for Gnosis Chain (xDAI) and Polygon (MATIC)
- Add deployment configurations for Honeyswap subgraphs
- Add support for Honeyswap subgraph on Polygon to create WETH pools with halved fees (trading/LP/protocol)
`Trades with ETH on Polygon have a fee of only 0.15%`
- (Currently WIP) Add staking reward handlers for native reward tokens (xCOMB/pCOMB) 
  - HoneyFarm contract documentation: https://hackmd.io/BFrhyOTUQ3O9REs5PuZahQ
  - Specific pools/LP tokens designated as reward bearing (not all earn reward tokens)
  - No Deposit/Withdraw events (Transfer emitted instead)
  - Pending rewards calculated similarly as Sushiswap Masterchef contract but expressed differently within HoneyFarm contract
- Currently deployed here:
  - Gnosis (xDAI): https://thegraph.com/hosted-service/subgraph/mrbrianhobo/honeyswap-xdai
  - Polygon (MATIC): https://thegraph.com/hosted-service/subgraph/mrbrianhobo/honeyswap-polygon?version=pending
Issues:
- `rewardTokenEmissionsAmount` seem pretty high (unsure if calculations are correct)

### CODE REVIEW

File: subgraphs/uniswap-forks/protocols/honeyswap/config/networks/matic/matic.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  getUntrackedPairs(): string[] {
+    return [];
+  }
+  getUntrackedTokens(): string[] {
+    return [];
+  }
+  getMinimumLiquidityThreshold(): BigDecimal {
+    return MINIMUM_LIQUIDITY_ONE_THOUSAND;
+  }
+}
```
Comment (steegecs) [inline subgraphs/uniswap-forks/protocols/honeyswap/config/networks/matic/matic.ts:101]: Leave an empty line at the end this file. And do it for all the others as well!
https://stackoverflow.com/questions/2402593/whats-the-reason-for-leaving-an-extra-blank-line-at-the-end-of-a-code-file#:~:text=It%20prevents%20you%20from%20forgetting,rid%20of%20your%20compiler%20warning.

Comment (steegecs) [reply subgraphs/uniswap-forks/protocols/honeyswap/config/networks/matic/matic.ts:101]: Everything else looks good to me so far for the swap portion. Nice job!

**Review by steegecs — COMMENTED** [review]
Looks good to me!

**Comment by this-username-is-taken** [comment]
Great work @mrbrianhobo! Thanks for helping out on the review @steegecs!

**Pull request merged**